src/main/cpp/t10_unique.cpp

```diff
@@ -25,5 +25,26 @@
 using namespace std;
 
 int t10_unique() {
+	int n, s, j;
+	j = 0;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 'j' is assigned a value that is never used.](https://app.codacy.com/app/thejerome/IFMO_CPP_programming_20190915/pullRequest?prid=4436470)